### DISCUSSION TITLE
SEB 3.1.1 won't work after allowing camera.

### DISCUSSION THREAD

**Discussion by augustojamarino** [SafeExamBrowser/seb-mac#260] [Q&A]
Hey guys, morning.

I try using the seb on Macos Ventura/Monterey with version seb 3.1.1 and latest 3.2.1 and camera wont working with allowed on seb.

Have a solution for this? may i change to windows or something?

**Comment by danschlet** [comment]
For web applications being able to access camera and microphone (and screen sharing), SEB must be configured correctly so it can use the modern WebView (WebKit2 browser engine). There are some technical restrictions from macOS (and iOS) by Apple, so supporting this may require some work from developers of exam solutions which have support for SEB built in. For example Moodle supports the modern WebView (and thus camera/microphone access) in newer versions (definitely in 4.1, for older versions Moodle admins might have to [install the plugin manually](https://github.com/ethz-let/moodle-quiz_accessrule_seb_js_api)).

What exam / learning management system are you using?

**Reply by danschlet** [reply]
Btw. technical details are explained in the [release notes](https://safeexambrowser.org/macosx/mac_release_notes_en.html) for SEB 3.0.

**Reply by augustojamarino** [reply]
Thanks Danschlet for reply,we use our system moddle,  i check with my team about this plugin its ok.

Thanks for information too about release notes.
